File: src/main/java/com/aws/primer/service/EventService.java
package com.aws.primer.service;

import java.util.List;

import com.aws.primer.dynamo.DailyEvents;
import com.aws.primer.dynamo.SiteNames;

public interface EventService {
    void saveEvent(DailyEvents eventDto);
    List<DailyEvents> getAllEvents();
    List<SiteNames> getAllSites();
}

File: src/main/resources/static/js/app/EventService.js
'use strict'

angular.module('awsprimerplc.services', []).factory('EventService',
		[ "$http", "CONSTANTS", function($http, CONSTANTS) {
			var service = {};			
			service.getAllEvents = function() {
				return $http.get(CONSTANTS.getAllEvents);
			}
			service.saveEvent = function(dailyEvents) {
				return $http.post(CONSTANTS.saveEvent, dailyEvents);
			}
			service.getAllSites = function() {
				return $http.get(CONSTANTS.getAllSites);
			}
			return service;
		} ]);